flax/nnx/nnx/compat/module.py

```diff
@@ -199,8 +199,8 @@ def is_initializing(self) -> bool:
 
     return self._object__state._initializing
 
-  def __init_subclass__(cls, experimental_pytree: bool = False) -> None:
-    super().__init_subclass__(experimental_pytree)
+  def __init_subclass__(cls, unsafe_pytree: bool = False) -> None:
```

Comment:
Can we add a link or docstring around here to explain this feature so that it's available somewhere in the API reference? I think this feature warrants explicit explanation (since its caveat is a bit hidden).